templates/cpp-template-default/proj.android/app/proguard-rules.pro

```diff
@@ -21,7 +21,7 @@
 -dontwarn org.cocos2dx.**
 -keep public class com.chukong.** { *; }
 -dontwarn com.chukong.**
--keep public com.huawei.android.** { *; }
```

Comment:
it seems like this change should be done in other files as well:
```
/Volumes/Untitled/android_build5/cocos2d-x $ git grep "com.huawei.android"
Binary file cocos/platform/android/java/libs/EngineDataManager.jar matches
templates/cpp-template-default/proj.android/app/proguard-rules.pro:-keep public class com.huawei.android.** { *; }
templates/cpp-template-default/proj.android/app/proguard-rules.pro:-dontwarn com.huawei.android.**
templates/js-template-default/frameworks/runtime-src/proj.android/app/proguard-rules.pro:-keep public com.huawei.android.** { *; }
templates/js-template-default/frameworks/runtime-src/proj.android/app/proguard-rules.pro:-dontwarn com.huawei.android.**
templates/lua-template-default/frameworks/runtime-src/proj.android/app/proguard-rules.pro:-keep public com.huawei.android.** { *; }
templates/lua-template-default/frameworks/runtime-src/proj.android/app/proguard-rules.pro:-dontwarn com.huawei.android.**
tests/cpp-empty-test/proj.android/app/proguard-rules.pro:-keep public com.huawei.android.** { *; }
tests/cpp-empty-test/proj.android/app/proguard-rules.pro:-dontwarn com.huawei.android.**
tests/cpp-tests/proj.android/app/proguard-rules.pro:-keep public com.huawei.android.** { *; }
tests/cpp-tests/proj.android/app/proguard-rules.pro:-dontwarn com.huawei.android.**
tests/game-controller-test/proj.android/app/proguard-rules.pro:-keep public com.huawei.android.** { *; }
tests/game-controller-test/proj.android/app/proguard-rules.pro:-dontwarn com.huawei.android.**
tests/js-tests/project/proj.android/app/proguard-rules.pro:-keep public com.huawei.android.** { *; }
tests/js-tests/project/proj.android/app/proguard-rules.pro:-dontwarn com.huawei.android.**
tests/lua-empty-test/project/proj.android/app/proguard-rules.pro:-keep public com.huawei.android.** { *; }
tests/lua-empty-test/project/proj.android/app/proguard-rules.pro:-dontwarn com.huawei.android.**
tests/lua-game-controller-test/project/proj.android/app/proguard-rules.pro:-keep public com.huawei.android.** { *; }
tests/lua-game-controller-test/project/proj.android/app/proguard-rules.pro:-dontwarn com.huawei.android.**
tests/lua-tests/project/proj.android/app/proguard-rules.pro:-keep public com.huawei.android.** { *; }
tests/lua-tests/project/proj.android/app/proguard-rules.pro:-dontwarn com.huawei.android.**
tests/performance-tests/proj.android/app/proguard-rules.pro:-keep public com.huawei.android.** { *; }
tests/performance-tests/proj.android/app/proguard-rules.pro:-dontwarn com.huawei.android.**
```

Reply:
Done https://github.com/cocos2d/cocos2d-x/pull/18804